i18n/card_games/ja.json

```diff
@@ -9,9 +9,12 @@
   "fish-checkbooks-title": "新しい束",
   "fish-m-card": "Qや10といった番号を言って尋ねましょう。",
   "fish-m-wait": "次のプレイヤー待ちです。",
+  "fish-turn-title": "次のターン",
   "fish-turn": "{0}のターン",
+  "fish-card-taken-title": "カードは取られました",
   "fish-card-taken": "{0} は {1} を尋ねました。 {2} 枚持っていたので渡さないといけません。",
   "fish-card-got": "{0} をもらいました。",
+  "fish-card-missed-title": "敵が尋ねました",
   "fish-card-missed-unsafe": "{0} は {1} を尋ねました。 {2} 枚も持っていないけど、大丈夫ではありません。",
   "fish-card-missed": "{0} は {1} を尋ねました。 {2} 枚持ってるので、大丈夫です。",
   "fish-fish-got": "{0} をもらいました! 頼んだものです!",
```

Comment:
Can't see them here, but the "fish-winners-sep" and "fish-winners-lastsep" strings should be removed.